src/liftwing_api/models/liftwing_model.py

```diff
@@ -0,0 +1,11 @@
+import requests
+
+class LiftwingModel:
+    def __init__(self, base_url):
+        self.base_url = base_url
+        # this base url will be used across every model
+
+    def request(self, endpoint, method="GET", data=None, headers=None):
+        # this method will make a request and return a json response
+        #return response.json()
+        return ""
```

Comment:
If we create an abstract class we can just issue a `pass` directive.
```suggestion
        pass
```

Reply:
I completed the method